doc/articles/contributing/guidelines/updating-dependencies.md

```diff
@@ -0,0 +1,57 @@
+# Guidelines for updating dependencies
+
+We use Dependabot to notify the team of any updates to dependencies. Once a week the robot will scan our dependencies and raise a pull-request if a new version is found. If an existing open pull-request is found for a dependency it will be closed and replaced with a new pull-request. The behaviour of the robot is [controlled by this configuration file](https://github.com/unoplatform/Uno/blob/master/.dependabot/config.yml). 
+
+## internal dependencies
+
+These dependencies don't change the public API surface and are typically safe to merge and we could potentially [configure mergify to automatically merge them if CI passes](https://medium.com/mergify/merging-bots-pull-requests-automatically-548ed0b4a424):
+
+- [CommonServiceLocator](https://github.com/unoplatform/uno/pull/1174)
```

Comment:
This specific dependency (`CommonServiceLocator`) should (MUST) be removed from Uno.

Reply:
ta